.github/workflows/jestTests.yml

```
@@ -0,0 +1,22 @@
+name: Jest and React Testing Library
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  Jest:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Install dependencies
+      working-directory: frontend
+      run: npm install
+
+    - name: Run tests
+      working-directory: frontend
```

It is possible to set the default working directory. Credits to @wettacked 
```suggestion
defaults:
  run:
    working-directory: ./frontend
    
jobs:
  Jest:
    runs-on: ubuntu-latest
    steps:
    - name: Checkout repository
      uses: actions/checkout@v2

    - name: Install dependencies
      run: npm install

    - name: Run tests
```